.github/workflows/ci.yml

```diff
@@ -62,4 +62,6 @@ jobs:
       - name: Install dependencies
         run: yarn --frozen-lockfile --prefer-offline
       - name: Build all projects
-        run: yarn workspaces run build
+        run: |
+          yarn workspaces run build
+          yarn workspaces run build-storybook
```

Comment:
We now run `build`, then `build-storybook` to ensure the presets are built first.